src/main/java/com/faforever/client/vault/replay/LiveReplayController.java

```diff
@@ -43,25 +46,35 @@
   private final UiService uiService;
   private final I18n i18n;
   private final MapService mapService;
+  private final TimeService timeService;
   public TableView<Game> liveReplayControllerRoot;
   public TableColumn<Game, Image> mapPreviewColumn;
+  public TableColumn<Game, Instant> startTimeColumn;
   public TableColumn<Game, String> gameTitleColumn;
   public TableColumn<Game, Number> playersColumn;
   public TableColumn<Game, String> modsColumn;
   public TableColumn<Game, String> hostColumn;
   public TableColumn<Game, Game> watchColumn;
 
-  public LiveReplayController(GameService gameService, UiService uiService, I18n i18n, MapService mapService) {
+  public LiveReplayController(GameService gameService, UiService uiService, I18n i18n,
+                              MapService mapService, TimeService timeService) {
     this.gameService = gameService;
     this.uiService = uiService;
     this.i18n = i18n;
     this.mapService = mapService;
+    this.timeService = timeService;
 
     selectedGame = new SimpleObjectProperty<>();
   }
 
   public void initialize() {
-    initializeGameTable(gameService.getGames().filtered(game -> game.getStatus() == GameStatus.PLAYING));
+    var gameSorter = Comparator.comparing(
```

Comment:
var got introduced in java 10 and only for local variables. 
To keep code consistent I would like to avoid var all together, also in favor of readability.